### PR TITLE
Update github-actions.md

### PR DESCRIPTION
**Pull request by skedwards88**
Addresses https://github.com/MicrosoftDocs/azure-devops-docs/issues/9815

### CODE REVIEW

File: docs/pipelines/ecosystems/github-actions.md [hunk 1]
```diff
@@ -85,21 +66,23 @@ If you have both Azure Pipelines and GitHub Actions workflows, you may want to t
...
     ```
 
-1. On the **Actions** page, verify that your workflow ran. Select the workflow title to see more information about the run. You should see a green check mark for the Azure Pipelines Action. Open the Action to see a direct link to the pipeline run. 
+1. Commit and push your workflow file.
+
+1. The workflow will run every time a commit is pushed to main or a pull request is opened against main. To verify that your action ran, open your GitHub repository and select **Actions**.
+
+    :::image type="content" source="media/github-actions-header.png" alt-text="Select Actions in the GitHub navigation"::: 
+
+1. Select the workflow title to see more information about the run. You should see a green check mark for the Azure Pipelines Action. Open the Action to see a direct link to the pipeline run.
```
Comment (skedwards88) [inline docs/pipelines/ecosystems/github-actions.md:75]: This last step is a bit out of date, but I am not sure how to add new screenshots. Here are more up-to-date screenshots/text as an example: https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/viewing-workflow-run-history. (This change just pushed last week, so it is pretty new.)

**Comment by PRMerger15** [comment]
@skedwards88 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/ecosystems/github-actions.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ If you have both Azure Pipelines and GitHub Actions workflows, you may want to t
 ## Prerequisites
 
 - A working Azure pipeline. [Create your first pipeline](../create-first-pipeline.md). 
-- A GitHub account with a repository. [Join GitHub](https://github.com/join) and [create a repository](https://docs.github.com/en/github/getting-started-with-github/create-a-repo). 
+- A GitHub account with a repository. [Join GitHub](https://github.com/join) and [create a repository](https://docs.github.com/github/getting-started-with-github/create-a-repo).
```
Comment (skedwards88) [inline docs/pipelines/ecosystems/github-actions.md:21]:
```suggestion
- A GitHub account with a repository. [Join GitHub](https://github.com/join) and [create a repository](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/create-a-repo). 
```

**Comment by PRMerger12** [comment]
@skedwards88 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by juliakm** [comment]
Thanks @skedwards88!

#sign-off

**Comment by ktoliver** [comment]
I can't view the new image reference in GitHub preview, but I see the named image file in the referenced folder in the repo, so we'll merge and verify it on the live site later today. 👍

**Pull request merged**